projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/case-flag/components/select-flag-location/select-flag-location.component.ts

```diff
@@ -77,8 +90,12 @@ export class SelectFlagLocationComponent implements OnInit {
     this.caseFlagsConfigError = true;
     this.errorMessages = [];
     this.errorMessages.push(
-      {title: '', description: SelectFlagLocationErrorMessage.FLAGS_NOT_CONFIGURED, fieldId: 'conditional-radios-list'});
+      { title: '', description: SelectFlagLocationErrorMessage.FLAGS_NOT_CONFIGURED, fieldId: 'conditional-radios-list' });
     // Return case flag field state and error messages to the parent
     this.caseFlagStateEmitter.emit({ currentCaseFlagFieldState: CaseFlagFieldState.FLAG_TYPE, errorMessages: this.errorMessages });
   }
+
+  ngOnDestroy(): void {
```

Comment:
@Kartheek-S - Need to add `public` before the method name. Also, it's worth checking the subscription actually exists before calling `unsubscribe()`, to avoid any `undefined` or `null` errors